snuba/cli/optimize.py

```diff
@@ -96,7 +96,9 @@ def optimize(
     last_midnight = (datetime.now() + timedelta(minutes=10)).replace(
         hour=0, minute=0, second=0, microsecond=0
     )
-    cutoff_time = last_midnight + settings.OPTIMIZE_JOB_CUTOFF_TIME
+
+    # add 1 hour to make the reddit TTL past the optimize job cuttoff time
+    cutoff_time = last_midnight + settings.OPTIMIZE_JOB_CUTOFF_TIME + timedelta(hours=1)
```

Comment:
Rename cutoff_time to redis_expire_time to make it more explicit that the variable is only used for determining redis expire time

Reply:
renamed